src/Skylelo/BetterCosmetics/Main.php

```diff
@@ -22,6 +22,7 @@ class Main extends PluginBase implements Listener
 
     public $bcParticles = [];
     private static $instance = null;
+    public $cfq;
```

Comment:
Typo? Should it not be `$cfg`?
```suggestion
    public $cfg;
```
Why is it public? 
```suggestion
    private $cfg;
```

The only use of `$cfg` is in https://github.com/Skylelo/BetterCosmetics/blob/f869d279e632dc6dfd9c032a4551c651a6689331/src/Skylelo/BetterCosmetics/PlayerJoinEvent.php#L34 which is in a **separate pull request**